### PR TITLE
Add sync tests integration

### PR DESCRIPTION
**Pull request by karimodm**

### CODE REVIEW

File: Nightly-Tests/Sync-Tests/createCluster.sh [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+K8S_NAMESPACE=$(kubectl config get-contexts $(kubectl config current-context) | tail -n+2 | awk '{print $5}')
+base_dir=$(pwd)
+IMAGE=${SYNC_IMAGE:-"iotacafe/iri-dev"}
+DATE=$(date '+%Y-%m-%d')
+
+if [ ! -d tiab ]; then
+  git clone --depth 1 https://github.com/iotaledger/tiab tiab
+fi
+ 
+virtualenv -p python2 venv
```
Comment (GalRogozinski) [inline Nightly-Tests/Sync-Tests/createCluster.sh:37]: python 2?

**Review by GalRogozinski — COMMENTED** [review]
I suppose it is fine
Only did a quick overview, so it is better that you do a cross-review

We need to remember that once we implement STING in IRI we can do w/o the milestone logic

File: Nightly-Tests/Sync-Tests/createCluster.sh (hunk 1 above)
Comment (karimodm) [reply Nightly-Tests/Sync-Tests/createCluster.sh:37]: Yeah, TIAB is currently only compatible with Python2 :/. There is an issue open on the repo already to port it to 3

**Comment by karimodm** [comment]
@galrogo This has been tested extensively on BuildKite, all working, please approve and merge ;)

**Pull request merged**